contents/security/passwordStorage.md

```diff
@@ -117,15 +117,15 @@ One question that is commonly asked by developers is where to store the salt. Th
 
 Many good password hashing algorithms today have built-in salts, such as [Argon2](https://github.com/P-H-C/phc-winner-argon2), [SCrypt](https://passlib.readthedocs.io/en/stable/lib/passlib.hash.scrypt.html) and [bcrypt](https://security.stackexchange.com/questions/4781/do-any-security-experts-recommend-bcrypt-for-password-storage). A good password hashing algorithm or library will salt automatically.
 
-### What if there is a server breach?
+### What if There is a Server Breach?
```

Comment:
```suggestion
### What If There Is a Server Breach?
```

Reply:
If is a conjunction and therefore I don't think it should be capitalized.

Reply:
Sure, I'm flexible on this. Let's go with yours 👍